lib/main.dart

```diff
@@ -11,31 +12,17 @@ void main() async {
 class MyApp extends StatelessWidget {
   @override
   Widget build(BuildContext context) {
-    return MaterialApp(
-      theme: ThemeData(
-        primarySwatch: Colors.blue,
-      ),
-      home: MyHomePage(),
-    );
-  }
-}
-
-class MyHomePage extends StatelessWidget {
-  @override
-  Widget build(BuildContext context) {
-    return Scaffold(
-      appBar: AppBar(
-        title: FutureBuilder<PackageInfo>(
-            future: PackageInfo.fromPlatform(),
-            builder: (context, snapshot) {
-              return snapshot.hasData
-                  ? Text(snapshot.data?.appName ?? "")
-                  : SizedBox.shrink();
-            }),
-      ),
-      body: Center(
-        child: Text(FlutterConfig.get('SECRET')),
+    return CupertinoApp(
```

Comment:
You can technically run a Cupertino app on either Android or iOS, but (due to licensing issues) Cupertino won't have the correct fonts on Android. For this reason, use an iOS-specific device when writing a Cupertino app. I think  Material is better.

Reply:
we use custom fonts now, is that still an issue? Btw, we mostly implement custom widgets so that's not a problem between using cupertino or material I think @doannimble

Reply:
No issue with custom font. I just wonder why you use Cupertino instead of Material?
As I can see the Material is more popular than Cupertino , even in the 3rd library, they usually use Material Theme :D
@luongvo